build/check.js

```diff
@@ -23,14 +23,23 @@ import path from 'path';
 import conf from './conf';
 
 /**
- * Checks whether codebase is in a state that is ready for submission. This means that code
- * follows the style guide, it is buildable and all tests pass.
+ * Builds Dashboard and ensures that the following requirements are met:
+ *   * the code follows the style guidelines.
+ *   * unit tests in frontend & backend should be successful.
```

Comment:
Can you improve formatting here? I.e., Make all bullets sentences (start with Capital letter and finish with dot) and use "is/are" instead of "should be". This will be easier to read :)